pkgs/applications/misc/solaar/default.nix

```diff
@@ -4,26 +4,27 @@
 , gtk3
 , python3Packages
 , wrapGAppsHook
-, gdk-pixbuf
 , libappindicator
-, librsvg
 }:
 
 # Although we copy in the udev rules here, you probably just want to use
-# logitech-udev-rules instead of adding this to services.udev.packages on NixOS
+# `logitech-udev-rules`, which is an alias to `udev` output of this derivation,
+# instead of adding this to `services.udev.packages` on NixOS,
 python3Packages.buildPythonApplication rec {
   pname = "solaar";
-  version = "1.1.3";
+  version = "1.1.4";
 
   src = fetchFromGitHub {
     owner = "pwr-Solaar";
     repo = "Solaar";
     rev = version;
-    hash = "sha256-6z22MnhUL9Da3G7UDmZsBAi2gHLNpiFEwe+pAtnP91s=";
+    hash = "sha256-nDfVF7g0M54DRpkH1rnZB8o+nCV4A6b1uKMOMRQ3GbI=";
   };
 
-  nativeBuildInputs = [ wrapGAppsHook gdk-pixbuf ];
-  buildInputs = [ libappindicator librsvg ];
```

Comment:
Both gdk-pixbuf and librsvg are required. I validated that removing either one results in the "Found an icon but could not load it. Most likely gdk-pixbuf does not provide SVG support." error.

Reply:
Oops, the icon works fine in my environment. Maybe because I'm on swaywm?
Reverted now.